### PR TITLE
Positions service controller

### PR DESCRIPTION
**Pull request by AnshulShirude**
## Description/Problem

Closes [story](copy_trello_card_link_here)

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.
List any dependencies that are required for this change.

The routes for the position were not done.

## Solution

What did you change?
I added in the routes for the position. I also updated the schema to now contain a department in a Position. Employees also contain a position now.

## Dependencies

[ ] This PR adds new dependencies

## Testing

Please describe how you tested this PR (manually and/or automatically)
Provide instructions so we can reproduce, and include screenshots of UI changes.

Tests don't completely pass. Working on trying to fix that since we updated the schema. The employee tests are messed up, but I tested the positions routes locally and it works.

- Manual Tests?
  - Add details and expected results here
- Unit/Integration/E2E Tests?
  - Add details and expected results here

### CODE REVIEW

File: apps/server/src/positions/entities/position.entity.ts [hunk 1]
```diff
@@ -1 +1,41 @@
...
+  @ApiProperty()
+  single: boolean;
+
+  @Exclude()
+  createdAt: Date;
+
+  @Exclude()
+  updatedAt: Date;
+
+  @Exclude()
```
Comment (elvincheng3) [inline apps/server/src/positions/entities/position.entity.ts:24]: can we move the department and signature fields above the created/updated at fields?

File: apps/server/src/positions/positions.service.ts [hunk 2]
```diff
@@ -1,22 +1,81 @@
...
+   * Create a new position.
+   * @param createPositionDto create position dto
+   * @returns the created position, hydrated
+   */
+  async create(createPositionDto: CreatePositionDto) {
+    const newPosition = await this.prisma.position.create({
+      data: {
+        name: createPositionDto.name,
+        departmentId: createPositionDto.departmentId,
+        // employeeId: createPositionDto.employeeId,
```
Comment (elvincheng3) [inline apps/server/src/positions/positions.service.ts:20]: comments

File: yarn.lock [hunk 3]
```diff
@@ -6614,21 +6624,26 @@ dot-case@^3.0.4:
     no-case "^3.0.4"
     tslib "^2.0.3"
 
+dotenv-expand@10.0.0:
```
Comment (elvincheng3) [inline yarn.lock:6627]: do we need to change the dependencies here?

Comment (AnshulShirude) [reply yarn.lock:6627]: No, removed them.

**Review by elvincheng3 — APPROVED** [review]

**Comment by elvincheng3** [comment]
[Positions service/controller](https://trello.com/c/E9kdP44r/35-positions-service-controller)

**Pull request merged**